### ISSUE TITLE
ERTManager.invert() --> error: malloc(): unsorted double linked list corrupted

### ISSUE THREAD

**Issue by emgema** [gimli-org/gimli#729]
![Bildschirmfoto vom 2024-05-27 12-38-12](https://github.com/gimli-org/gimli/assets/86672926/7c838645-2163-4368-a573-8ebe00cf85fa)

I want to perform a ERT-inversion, 
but program fails at mgr.invert(), more precise 
in ertModelling.py, line 178 in response: "resp = self._core.response(mod)"

with error:
"Fatal Python error: Aborted"  (ipython)
"malloc(): unsorted double linked list corrupted" (when started inside bash)

I use:  
![Bildschirmfoto vom 2024-05-27 12-39-54](https://github.com/gimli-org/gimli/assets/86672926/d456d511-6286-4d71-9596-775e23698be5)
on Linux Mint 21.3 Cinnamon

thanks for your help!

**Comment by emgema** [comment]
It might be related that I solved this issue #721 by
```
ln -s libcholmod.so.5 libcholmod.so.3
ln -s libumfpack.so.6 libumfpack.so.5
```
just after installation...

**Comment by halbmy** [comment]
This might explain a lot. Linking to a different version can be a solution but it is not recommended. Keep updated with #721.

**Comment by emgema** [comment]
ok, thanks! Until a fix of #721 I use:
![Bildschirmfoto vom 2024-05-27 17-02-36](https://github.com/gimli-org/gimli/assets/86672926/87217c9e-e95d-48a7-8091-c5feb3abefae)
